.travis.yml

```diff
@@ -543,3 +543,4 @@ jobs:
 branches:
   only:
     - master
+    - release-6.10.0
```

Comment:
Actually this is fine to leave in this branch, as long as we delete on the merge back to master.

Reply:
Welp. The change is after this.